docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
```

Comment:
This is the guy that cause me much time and debugging.
As is, this causes as error on my machine when setting up the postgresql database: `could not translate host name "db" to address: Name or service not known`.

https://www.linux.com/learn/docker-volumes-and-networks-compose: suggests that named volumes are used instead. Bind volumes can be used if you want to bind a specific file into the docker container. (https://stackoverflow.com/questions/41637505/how-to-persist-data-in-a-dockerized-postgres-database-using-volumes)

So, we can change this to `db-data:/var/lib/postgresql/data`, for example. Or any other name. And then at the bottom of the file add:
```yml
volumes:
  db-data:
```

Reply:
This works.